src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+  const calculator = {
+    operate(callback, number) {
+      if (typeof callback === 'function') {
+        result = callback(result, number);
+      }
+
+      return this;
+    },
+
+    getResult() {
+      return result;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
```

Comment:
Similar to the `add` method, the `subtract` method returns the difference of `a` and `b` without updating the internal `result`. Consider updating the `result` variable if that is the intended functionality.